.github/ISSUE_TEMPLATE/question-query-template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Question/Query template
+about: Use this template to file a question/query for the SDK team
+title: "[QUERY]"
+labels: ''
+assignees: ''
+
+---
+
+**Query/Question**
+A clear and concise ask/query.
+
+***Why is this not a Bug or a feature Request?***
+A clear explanation of why is this not a bug or a feature request?
+
+**Setup (please complete the following information if applicable):**
+ - OS: [e.g. Android 10]
+ - IDE : [e.g. Android Studio 4.0 ]
```

Comment:
Same here.

Reply:
Resolved the same as here: https://github.com/Azure/azure-notificationhubs-android/pull/111#discussion_r466600891